projects/m1/032-taxi-fare/js/index.js

```diff
@@ -0,0 +1,10 @@
+function myTaxiFare (distance) {
+    const baseFare = 4;
+    let distanceInMeters = (distance * 1000) / 140;
+    let totalFare = Math.floor(distanceInMeters * 0.25) + baseFare;
+    return totalFare;
+  }
+  
+  let distance = prompt('Insert the distance in kilometers');
+  let fare = myTaxiFare (distance);
+  console.log(`Total fare: ${fare} $`);
```

Comment:
Pay attention to all (four) of the `lets that you have used.
Are they needed, or was it better to use `const`?! ;-)

Reply:
I think all the four LETs are used correctly becouse  the values contained in them could be modified and not remain constant during execution

Reply:
Could you show me the row number where the values ​​are changed?
If they are **modified** it is correct to use **let**, otherwise, the **risk** is an **inadvertent modification** in the future. ;-)
Bad practice ;-)

Reply:
Could the reason for the value change be due to the prompt that defines the type of value we give to the variable? Or am I mistaken? Also referring to the following exercise.

Reply:
You have to consider the **variation during the single execution** of the script ;-)
Once the **script finishes** its execution, the **variables** are **deleted**.
If you want to learn more about memory management:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Memory_management